### PR TITLE
refactored counter_test to use assert statements and renamed variable…

### PR DESCRIPTION
**Pull request by conwaychriscosmo**
…s for consistency

/kind cleanup

**What this PR does / why we need it**:
The PR refactors a test to be more readable.  It is unnecessary but nice.

**Special notes for your reviewer**:
It is my first code change.
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

**Comment by conwaychriscosmo** [comment]
@mtaufen @RainbowMango @logicalhan @yastij Please let me know your thoughts and approve my PR.

**Comment by conwaychriscosmo** [comment]
\wg component-standard

File: staging/src/k8s.io/component-base/metrics/counter_test.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package metrics
 import (
 	"bytes"
 	"testing"
+	"github.com/stretchr/testify/assert"
```
Comment (RainbowMango) [inline staging/src/k8s.io/component-base/metrics/counter_test.go]: nit: Imports should be grouped. standard libs -> third party -> k8s imports

**Comment by RainbowMango** [comment]
@conwaychriscosmo 
Thanks for your contribution.

> The PR refactors a test to be more readable. It is unnecessary but nice.

Would you mind introduce `github.com/stretchr/testify/assert` to all of the tests under this package?

File: staging/src/k8s.io/component-base/metrics/counter_test.go (hunk 1 above)
Comment (yastij) [reply staging/src/k8s.io/component-base/metrics/counter_test.go]: +1

**Review by yastij — COMMENTED** [review]
mostly lgtm, one nit on the imports
/ok-to-test

**Comment by conwaychriscosmo** [comment]
> @conwaychriscosmo
> Thanks for your contribution.
> 
> > The PR refactors a test to be more readable. It is unnecessary but nice.
> 
> Would you mind introduce `github.com/stretchr/testify/assert` to all of the tests under this package?

I was hoping that people would want that :)

**Review by mtaufen — COMMENTED** [review]
Looks correct to me.

Please run gofmt on your changes.

File: staging/src/k8s.io/component-base/metrics/counter_test.go (hunk 1 above)
Comment (mtaufen) [reply staging/src/k8s.io/component-base/metrics/counter_test.go]: @conwaychriscosmo, as @RainbowMango mentioned it's conventional in the K8s project to separate each group of imports with a blank line. In this case, the assert lib should be in the middle group, along with semver and expfmt.

**Comment by RainbowMango** [comment]
> Looks correct to me.
> 
> Please run gofmt on your changes.

Yes, Run `./hack/update-gofmt.sh` in the kubernetes directory.

**Comment by RainbowMango** [comment]
@conwaychriscosmo 
Would you mind fix the comments and squash your commits?

**Comment by conwaychriscosmo** [comment]
@RainbowMango 
My mistake not properly addressing the comments, moving an import statement two lines isn't hard; I think I accidentally reverted the changes that addressed them when I amended the commit with a new author. I would rather not squash the commits because the documentation I read about testing and contributing said squashing commits was optional.

**Comment by logicalhan** [comment]
> I would rather not squash the commits because the documentation I read about testing and contributing said squashing commits was optional.

I would also prefer that you squash the commits.

**Review by logicalhan — APPROVED** [review]
/lgtm

**Pull request merged**